### PR TITLE
added property to show inactive titles

### PR DESCRIPTION
**Pull request by agam1092005**
Issue #68  resolved
Added new feature: Added property to show inactive titles using MediaQuery width properties to fit all the titles in the screen/display size

Updated readme file for docs
Signed-off-by: Agampreet Singh <agam1092005@gmail.com>

### CODE REVIEW

**Review by antonio-pedro99 — CHANGES_REQUESTED** [review]
@agam1092005 thanks for the PR. 
Can you add these changes to the CHANGELOG.md file?

File: CHANGELOG.md
Comment (antonio-pedro99) [inline CHANGELOG.md:1]: @agam1092005 can you add under version 7.0.0? That's the next version.

Comment (agam1092005) [reply CHANGELOG.md:1]: Done 👍🏻

**Review by antonio-pedro99 — APPROVED** [review]
LGMT
@agam1092005 thanks!

**Pull request merged**